dts/common/yaml/i2c.yaml

```diff
@@ -0,0 +1,25 @@
+---
+title: I2C Base Structure
+id: i2c
+version: 0.1
+
+description: >
+    This binding gives the base structures for all I2C devices
+
+properties:
+  - clock-frequency :
+      type: int
+      category: optional
+      description: Desired I2C bus clock frequency in Hz
```

Comment:
In properties description, I'm not opposed to clarifying driver behavior as it could diverge in some cases vs Linux's driver behavior and does not have impact on dts files (and won't cause any divergence on dts files themselves)